src/js/models/documentexports.js

```diff
@@ -67,6 +67,44 @@ define(function (require, exports, module) {
     }));
 
     /**
+     * Generate List of ExportAssets based on an array of simple props objects
```

Comment:
The following two functions were factored directly out of `fromDescriptors` below.